### ISSUE TITLE
10$: 'new await();' in global scope gives false error

### ISSUE THREAD

**Issue by strager** [quick-lint/quick-lint-js#708]
```js
class await {
  constructor() { console.log('wtf'); }
}

new await();
```
qljs falsely reports 'E0078: missing expression between parentheses'
